space.rirusha.Cassette.yml

```diff
@@ -0,0 +1,51 @@
+id: space.rirusha.Cassette
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.vala
+command: cassette
+finish-args:
+- --share=network
+- --share=ipc
+- --socket=fallback-x11
+- --device=dri
+- --socket=pulseaudio
+- --socket=wayland
+- --filesystem=xdg-music:ro
+- --filesystem=xdg-download:ro
+- --talk-name=org.mpris.MediaPlayer2.Player
+- --own-name=org.mpris.MediaPlayer2.cassette
+build-options:
+  append-path: /usr/lib/sdk/vala/bin
+  prepend-ld-library-path: /usr/lib/sdk/vala/lib
+cleanup:
+- /include
+- /lib/pkgconfig
+- /man
+- /share/doc
+- /share/gtk-doc
+- /share/man
+- /share/pkgconfig
+- /share/vala
+- '*.la'
+- '*.a'
+modules:
+- name: blueprint-compiler
+  buildsystem: meson
+  sources:
+  - type: git
+    url: https://gitlab.gnome.org/jwestman/blueprint-compiler
```

Comment:
missing the cleanup rule to remove all of this build only dependency.

Reply:
My bad, missed that point in the documentation. Thanks for the correction